### PR TITLE
Update cqrs-microservice-reads.md

### PR DESCRIPTION
**Pull request by aymanizz**
## Summary

Ordering microservice changed to use static ViewModels instead of dynamic ones in [this PR](https://github.com/dotnet-architecture/eShopOnContainers/pull/380).

### CODE REVIEW

**Comment by aymanizz** [comment]
If this is in reference to the implementation in the docs (not the latest implementation on the repo) then this change might not be needed (besides there is already a tip explaining the change.)

If that's the case, I do think that the wording could be improved to something like `as implemented below`?

File: docs/architecture/microservices/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ Since the queries are performed to obtain the data needed by the client applicat
 
 The returned data (ViewModel) can be the result of joining data from multiple entities or tables in the database, or even across multiple aggregates defined in the domain model for the transactional area. In this case, because you are creating queries independent of the domain model, the aggregates boundaries and constraints are completely ignored and you're free to query any table and column you might need. This approach provides great flexibility and productivity for the developers creating or updating the queries.
 
-The ViewModels can be static types defined in classes. Or they can be created dynamically based on the queries performed (as is implemented in the ordering microservice), which is very agile for developers.
+The ViewModels can be static types defined in classes (as is implemented in the ordering microservice.) Or they can be created dynamically based on the queries performed, which is very agile for developers.
```
Comment (IEvangelist) [inline docs/architecture/microservices/microservice-ddd-cqrs-patterns/cqrs-microservice-reads.md:30]:
```suggestion
The ViewModels can be static types defined in classes (as is implemented in the ordering microservice). Or they can be created dynamically based on the queries performed, which is very agile for developers.
```

**Review by IEvangelist — CHANGES_REQUESTED** [review]
Corrected the placement of the `.`

**Comment by BillWagner** [comment]
@IEvangelist I think your requests have been addressed. Can you check, and merge if so?

**Review by IEvangelist — APPROVED** [review]
Looks good to me, we'll :shipit: - thank you 🙏

**Pull request merged**